README.rst

```diff
@@ -404,6 +404,10 @@ TMT_TEST_DATA
     back from the guest and available for inspection after the
     test execution is finished.
 
+TMT_TEST_SERIALNUMBER
```

Comment:
```suggestion
TMT_TEST_SERIAL_NUMBER
```

Let's use underscore to separate all words to make the variable name consistent. We want to rename the variable as well anyway.

Reply:
Fixed in 6292e190